src/python_inspector/utils_pypi.py

```diff
@@ -107,8 +107,12 @@
     "39": "3.9",
     "310": "3.10",
     "27": "2.7",
+    "311": "3.11",
 }
 
+valid_python_versions = list(PYTHON_DOT_VERSIONS_BY_VER.keys())
+valid_python_versions.extend([dot_ver for pyver, dot_ver in PYTHON_DOT_VERSIONS_BY_VER.items()])
+
```

Comment:
Why did you make this change in these two lines?

Reply:
Actually I didn't made these changes, these changes automatically gets added to my PR after I ran git pull command and changed the head to different position, but will remove these for now.

Reply:
https://github.com/nexB/python-inspector/pull/109/commits/5a2ceb2e7948f0d9d8e6f564a72c4dc7b55dcb1f Removed this stuff which was not part of this PR